src/object-properties.js

```diff
@@ -66,6 +66,7 @@ const properties = {
       colorExpression: "",
     },
   },
+  components: [],
```

Comment:
This is probably a good addition. However when we add the logic for resolving everything we need to make sure that we also add tests for a missing components array

Reply:
I will add that scenario to the migration test. But there is another unit test included in the second PR that take the missing components into consideration.